### PR TITLE
Changed fabric.mod.json

### PR DESCRIPTION
**Pull request by DragonEggBedrockBreaking**
Changed fabric.mod.json to make it so that carpet autocraftingtable displays as a sub-part of carpet in modmenu, as Carpet TIS Additions has already done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
-# carpet-autoCraftingTable
-autoCraftingTable extension for carpet-mod. Implementation based on Skyrising's implementation for autoCraftingTable for QuickCarpet - implemnted here as an extension for fabric-carpet mod.
+# Carpet AutoCraftingTable - Unofficial Fork
+
+### IF YOU WANT THE MOD, GO HERE: https://github.com/gnembon/carpet-autoCraftingTable
```
Comment (gnembon) [inline README.md:3]: que?

Comment (DragonEggBedrockBreaking) [reply README.md:3]: Sorry, I made the same mess up on multiple mods... I forgot about the pull request and randomly edited the main branch. Imma undo the commit and temporarily move it to another branch (on my fork) which is unrelated to this pr. I will comment here when I'm done.

Comment (DragonEggBedrockBreaking) [reply README.md]: Fixed it now. Sorry....

**Comment by DragonEggBedrockBreaking** [comment]
Will be refactored

**Pull request closed without merging**